src/chat/__tests__/chatReducers-test.js

```diff
@@ -99,6 +99,68 @@ describe('chatReducers', () => {
       expect(newState).toEqual(expectedState);
       expect(newState).not.toBe(initialState);
     });
+
+    describe('if new message key is not present than create new key', () => {
+      test('if message is of type stream, then create key of topicNarrow type', () => {
+        const initialState = deepFreeze({
+          messages: {
+            [JSON.stringify(topicNarrow('all', 'GCI'))]: [{ id: 1 }, { id: 2 }],
```

Comment:
Use better values. For example `some stream` and `some topic` make it much more obvious what value is what.